src/main/resources/application-aws.yml

```diff
@@ -1,36 +1,32 @@
-environment:
-  domain: backbase.test:8080
-  domain-identity: backbase.test:8180
 
 bb-fuel:
   platform:
-    infra: http://${environment.name}-${infra.base:infra}.${environment.domain}
-    identity: http://${environment.name}-bbiam.${environment.domain-identity}
-    registry: ${bb-fuel.platform.infra}/registry/eureka
-    gateway: ${bb-fuel.platform.infra}/gateway/api
-    auth: ${bb-fuel.platform.gateway}/auth/login
-    tokenconverter: ${bb-fuel.platform.infra}/bb-authentication-token-converter-service
+    registry: https://registry.${environment.name}.${environment.domain}
+    identity: https://identity.${environment.name}.${environment.domain}
+    gateway: https://app.${environment.name}.${environment.domain}/api
+    auth: https://app.${environment.name}.${environment.domain}/api/auth/login
```

Comment:
This comments is about `registry` and `auth`. Shall we leave those since this file is aimed to BaaS and in BaaS these 2 components are missing? I'm not sure @cyberlad24 if the tool works anyway if we remove this